python/src/schema.rs

```diff
@@ -1008,14 +1008,69 @@ impl PySchema {
     /// Return equivalent PyArrow schema
     ///
     /// :rtype: pyarrow.Schema
-    #[pyo3(text_signature = "($self)")]
-    fn to_pyarrow(self_: PyRef<'_, Self>) -> PyResult<PyArrowType<ArrowSchema>> {
+    #[pyo3(text_signature = "($self, as_large_types)")]
```

Comment:
I think this should handle the default value:
```suggestion
    #[pyo3(signature = "(as_large_types=false)")]
```